### PR TITLE
DDF-1910: updated Github page to include documentation links.

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?
Add documentation links to gh-pages branch. 

#### Who is reviewing it?
@stustison @pklinef @Lambeaux @vinamartin @mcalcote @shaundmorris 

#### How should this be tested?
Manual navigation of pages.

#### Any background context you want to provide?
This should be merged with the `gh-pages` branch, not `master`.

#### What are the relevant tickets?
DDF-1910

#### Screenshots (if appropriate)

documentation-page

#### Checklist:
- [X] Documentation Updated (sorta)
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by Lambeaux** [comment]
@ricklarsen I really like the included screenshot. That looks **really** sharp! Clear and easy to understand.

File: Downloads.html [hunk 1]
```diff
@@ -52,7 +53,7 @@
                 </div>
                 <div class="col-md-4">
                     <h2>Quick Start Guide</h2>
-                    <p>The DDF install guide and documentation are located on the wiki site.</p>
+                    <p>Get a sample or test installation up quickly.</p>
```
Comment (vinamartin) [inline Downloads.html]: I don't have any suggestions, but I found myself having to read this phrase a couple times in order to understand it. Thoughts?

**Comment by vinamartin** [comment]
This will definitely be useful to have on the site! :shipit:

File: Downloads.html (hunk 1 above)
Comment (ricklarsen) [inline Downloads.html]: Maybe "Basic instructions for a test or sample installation" ?

Comment (vinamartin) [inline Downloads.html]: Yeah that works for me :+1:

Comment (Lambeaux) [inline Downloads.html]: Rephrase: `Get an installation up quickly, regardless if sample or test.`

Comment (Lambeaux) [inline Downloads.html]: Oh it's almost like a title description. Maybe this is better: `Get an installation up quickly for sample or test.`

File: Documentation-versions.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            </div>
+
+            <div class="row">
+                <div class="col-md-3">
+                    <h2>Managing</h2>
+                    <p>Documentation for administrators installing or maintaining DDF</p>
+                </div>
+                <div class="col-md-3">
+                    <h2>Integrating</h2>
+                    <p>Documenation of services and architecture used to integrate DDF</p>
```
Comment (Lambeaux) [inline Documentation-versions.html]: typo

File: Documentation-versions.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                </div>
+                <div class="col-md-3">
+                    <h2>Extending</h2>
+                    <p>Documenation for developers interested in contributing to DDF</p>
```
Comment (Lambeaux) [inline Documentation-versions.html]: typo

File: Downloads.html (hunk 1 above)
Comment (Lambeaux) [inline Downloads.html]: `Get an installation up quickly for sample or test.`

**Comment by Lambeaux** [comment]
Aside from a few comments, LGTM :+1:

File: Documentation-versions.html (hunk 2 above)
Comment (ricklarsen) [inline Documentation-versions.html]: fixed.

File: Documentation-versions.html (hunk 3 above)
Comment (ricklarsen) [inline Documentation-versions.html]: fixed.

File: Documentation-versions.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+                            <td colspan="2"><a href="http://artifacts.codice.org//service/local/artifact/maven/redirect?g=ddf&a=docs&v=2.7.0&r=public&c=Export&e=zip">zip</a></td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+
+            <footer>
+                <div class="container">
+                    <div class="row">
+                        <div class="col-md-10">Website and software - <a href="http://www.codice.org" target="_blank">The Codice Foundation</a></div>
```
Comment (mcalcote) [inline Documentation-versions.html]: Not an actual problem, but for some reason I feel like the 's' in software should be capitalized. Either way -> Not a bug.

**Comment by mcalcote** [comment]
LGTM :+1:

**Comment by jlcsmith** [comment]
@ricklarsen does this PR include the javadoc fix?

**Comment by ricklarsen** [comment]
@jlcsmith Not yet, but as soon as that fix is working, it will.

**Comment by michaelmenousek** [comment]
zip is under the html column.  Rename that column to "download" perhaps?

**Comment by michaelmenousek** [comment]
### Managing DDF
How to install and configure, targeted for system administrators

### Integrating DDF
How to connect to and from DDF using service interfaces, targeted for system integrators

### Extending DDF
How to add and modify system behaviors, targeted for developers

**Comment by shaundmorris** [comment]
@jlcsmith after talking with @ricklarsen decided to go ahead with this PR. There is still an existing PR on fixing the javadoc generation and still need to look at deploying. After that is working there will be another PR to update the site appropriately.

**Pull request merged**